src/playwright/test.ts

```diff
@@ -11,22 +11,23 @@ import { AxeBuilder } from '@axe-core/playwright';
 import customElements from '../../custom-elements.json' with { type: 'json' };
 import { v8CoverageAttachmentName } from './constants.js';
 
-// Anything the comes from the browser will get serialized before it gets sent
-// back to tests, which run in Node.js. So `addEventListener()` can only return
-// a subset of event properties. But these should cover most of our use cases.
-interface SimpleEvent {
+interface SerializableEvent {
   bubbles: boolean;
   cancelable: boolean;
   composed: boolean;
   defaultPrevented: boolean;
-  detail?: unknown;
-  target: string | null;
```

Comment:
Unused for now. We can add to this type are use cases for more properties prevent themselves.